### ISSUE TITLE
Version 4.2.2 is still version 4.2.1 on NPMJS

### ISSUE THREAD

**Issue by DerkSchooltink** [codekie/openapi-examples-validator#130]
Installing the newest 4.2.2 release does not include the latest changes regarding the omission of the additional properties flag. 

Package: [NPMJS](https://www.npmjs.com/package/openapi-examples-validator/v/4.2.2)

Version output:

```
$ npm uninstall openapi-examples-validator -g
$ npm install openapi-examples-validator@4.2.2 -g
$ openapi-examples-validator --version
4.2.1
```

Initially I thought this might be an installation issue, but after targeting latest and 4.2.2, I seem to get the same behaviour. 

Tested by building a local version of the validator, which works correctly. Somehow it seems that the wrong version is used on NPMJS (and also Dockerhub, it also targets 4.2.2 but also outputs version 4.2.1).

**Comment by codekie** [comment]
Yes, because it's not released yet.

**Comment by DerkSchooltink** [comment]
> Yes, because it's not released yet.

Perhaps I'm misunderstanding but according to NPMJS the version exists. What exactly does that number mean if it doesn't contain the actual fixes/features from that version?

![image](https://user-images.githubusercontent.com/22047701/107535563-221b3280-6bc1-11eb-9824-cd30608dfea7.png)

**Comment by codekie** [comment]
When it comes to the included changes within a release, refer to the [changelog](https://github.com/codekie/openapi-examples-validator/blob/master/CHANGELOG.md)

**Comment by codekie** [comment]
@DerkSchooltink The newest version (`v4.3.1`), including your feature, has been published. I looked into the `--version`-issue and yes, the output was wrong, while the content was right. Thanks for pointing that out.

As for the explanation: I use a module, called [standard-version](https://www.npmjs.com/package/standard-version), to automatically bump the package-version and generate the changelog, based on the commit-messages (see also: [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)). However, in the release-step, the module was built with the package-version (but the new changes) before the version was bumped by `standard-version`. This was why `openapi-examples-validator --help` showed the wrong, previous version-number. I modified that release-step to re-build the module again, after the version has been bumped, so now `--help` shows the right version.
